src/__tests__/pages/beta/[viewType]/programmes/[programmeSlug]/units/unitListing.test.tsx

```diff
@@ -4,7 +4,7 @@ import curriculumApi from "../../../../../../../node-lib/curriculum-api/__mocks_
 import UnitListingPage, {
   getStaticPaths,
   getStaticProps,
-} from "../../../../../../../pages/[viewType]/programmes/[programmeSlug]/units";
+} from "../../../../../../../pages/teachers/programmes/[programmeSlug]/units";
```

Comment:
please start replacing these with aliases

Reply:
Have done for a lot of pages in this PR, missed this one